docs/kiali/05-01-kiali.md

```diff
@@ -21,4 +21,4 @@ This table lists the configurable parameters, their descriptions, and default va
 | **deployment.accessibleNamespaces** | Specifies the Namespaces Kiali can access to monitor the Service Mesh components deployed there. You can provide the names using regex expressions. The default value is `**`(two asterisks) meaning Kiali can access any Namespace. | `**` |
 
```

Comment:
In regards to the config options above:
- metrics.enabled - I prefer to not mention as we have it disabled by default (I think it was not working proper)
- server.webroot - would you ever change it?

Instead check the other attributes of the kiali CRD which might be relevant like:
- resources - to increase memory limit for exampe
- kubernetes_config.qps - to adjust apiserver throttling rate

Reply:
I will check to add the extra configurations that might be useful.

On those to remove:
- `metrics.enabled`: will remove it.
- `server.webroot`: might be useful to use a dedicated path. In any case it is an option that exists and does not harm to document it I think.

Reply:
Removed the `metrics.enabled` and added the suggested config values.